proposals/README.md

```diff
@@ -0,0 +1,6 @@
+# Azure Kinect Sensor SDK Proposals
+
+New features proposals are living documents describing the current thinking about a given feature. Proposals are the way for Azure Kinect team  to communicate to the public upcoming new feature work and get the feedback.
+At this time, all proposals will be created by Microsoft Azure Kinect team for the most desired features, but if you have a great idea for a proposal, please submit it on [User Voice](https://feedback.azure.com/forums/920053).
+
+We would love your feedback on every proposal and therefore we will create a GitHub issue where you can prove it.
```

Comment:
Not sure what we are tying to say here

Reply:
@Greendogo, we picked Nvidia Jetson family devices for a few reasons:

- It is something that most of the users were asking on Customer voice

- it fits Azure Kinect DK both SDK's requirements including GPU usage, so the built in GPU capabilities will be supported.


For this proposal, we will focus on those 2 boards only and we will consider other types of devices if we see the demand for it.

Reply:
fixed it